### PR TITLE
added delete feature to notes

### LINKED ISSUE #5: User should be able to edit or delete own notes

### PR DESCRIPTION
**Pull request by cgisala**
@svenJonis138 @KelseyStiff  please check out the code for the note delete feature

### CODE REVIEW

File: lmn/static/js/confirm_delete.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// the form will be submitted as usual.
+
+var deleteButtons = document.querySelectorAll('.delete');
+
+deleteButtons.forEach(function(button){
+
+  button.addEventListener('click', function(ev){
+
+    // Show a confirm dialog
+    var okToDelete = confirm("Delete place - are you sure?");
```
Comment (claraj) [inline lmn/static/js/confirm_delete.js:16]: Place?

File: lmn/urls.py [hunk 2]
```diff
@@ -21,6 +21,8 @@
     path('notes/detail/<int:note_pk>/', views_notes.note_detail, name='note_detail'),
     path('notes/for_show/<int:show_pk>/', views_notes.notes_for_show, name='notes_for_show'),
     path('notes/add/<int:show_pk>/', views_notes.new_note, name='new_note'),
+    path('notes/<int:note_pk>/delete', views_notes.delete_note, name='delete_note'), #issue 5 delete note by chris
```
Comment (claraj) [inline lmn/urls.py]: As noted in the last pull request review, don't leave comments like this. Git tags every line of code you write with your name, and you should link it to the issue using this pull request with a comment in the *pull request* like "partially addresses #5"

File: lmn/views/views_notes.py [hunk 3]
```diff
@@ -44,3 +45,12 @@ def notes_for_show(request, show_pk):
 def note_detail(request, note_pk):
     note = get_object_or_404(Note, pk=note_pk)
     return render(request, 'lmn/notes/note_detail.html' , { 'note': note })
+
+# issue 5 delete note by chris
```
Comment (claraj) [inline lmn/views/views_notes.py]: Again this comment should be removed. Comments should explain how the code works

**Review by claraj — CHANGES_REQUESTED** [review]
I haven't run the code but someone on your team will do that. 
Remove the example images, those are not part of the code base
No tests - make sure you add those

**Comment by KelseyStiff** [comment]
@cgisala thanks for taking care of this, once you address the issues/conflicts above let us know and I'll take another look at it!

File: lmn/static/js/confirm_delete.js (hunk 1 above)
Comment (cgisala) [reply lmn/static/js/confirm_delete.js:16]: @KelseyStiff @svenJonis138 removed the example images.  Still working on the test

File: lmn/urls.py (hunk 2 above)
Comment (cgisala) [reply lmn/urls.py]: Removed the comment and linked pull request to the issue #5

File: lmn/views/views_notes.py (hunk 3 above)
Comment (cgisala) [reply lmn/views/views_notes.py]: @KelseyStiff @svenJonis138 removed the comment.

**Comment by cgisala** [comment]
Closing pull request due to creating a new branch.

**Pull request closed without merging**